### ISSUE TITLE
[Checkbox] Text-only radio choice

### ISSUE THREAD

**Issue by SunflowerFuchs** [fomantic/Fomantic-UI#822]
# Feature Request

## Description
My coworker suggested that it would be nice to have a radio-style selection of options, but consisting only of a block of text without a visible radio button, and instead the chosen option being highlighted (e.g. with a different background colour).

## Example
I quickly threw this together to have a basic example, it's not the prettiest but it works
https://jsfiddle.net/wa6tLnkr/

**Comment by GammaGames** [comment]
That would be a nice feature! I've been using `.ui.segment`s with the following rules to break up forms when there was nested options:

```css
#form_id .segment {
    transition-property: box-shadow;
    transition-duration: 0.1s;
    cursor: pointer;
}
#form_id .segment.active {
    color: #fff!important;
    background-color: #2185d0!important;
}
#form_id .segment:not(.disabled):hover {
    box-shadow: 0 1px 3px 0 #bcbdbd, 0 0 0 1px #d4d4d5;
}
```
![Screen record from 2019-06-19 08 43 53](https://user-images.githubusercontent.com/7832163/59775511-95df2d80-926e-11e9-9eb3-7198a4da4eb7.gif)

**Comment by lubber-de** [comment]
Implemented/Possible by #2773

**Comment by GammaGames** [comment]
It looks so good @lubber-de :star_struck:
